taxcalc/functions.py

```diff
@@ -188,17 +266,16 @@ def ItemDed(_posagi, e17500, e18400, e18425, e18450, e18500, e18800, e18900,
         c21040 = 0.0
 
 
-    if c21060 > _nonlimited and c00100 > _limitratio:
```

Comment:
The if statements here is redundant, because the previous if statement has already handled the issue.